requirements.txt

```diff
@@ -12,6 +12,7 @@ wand==0.4.4
 jsonschema==2.6.0
 numpy==1.13.3
 PyYAML==3.12
+mock==2.0.0
```

Comment:
don't need this - mock is in `unittest` in python 2.7 and above

Reply:
Done